### PR TITLE
WIFI-13041-CIG-WF660a-5.4-Kernel-Firmware-upgrade

### PR DESCRIPTION
**Pull request by Kenshi0000**

### CODE REVIEW

File: feeds/ipq807x_v5.4/ipq60xx/base-files/etc/board.d/02_network [hunk 1]
```diff
@@ -15,7 +15,6 @@ qcom_setup_interfaces()
 	hfcl,ion4xi|\
 	hfcl,ion4x|\
 	hfcl,ion4x_2|\
-	hfcl,ion4x_3|\
 	hfcl,ion4xe)
```
Comment (blogic) [inline feeds/ipq807x_v5.4/ipq60xx/base-files/etc/board.d/02_network:18]: this appears to be based on an old tree and removes stuff

File: feeds/ipq807x_v5.4/ipq60xx/base-files/etc/board.d/02_network [hunk 2]
```diff
@@ -34,12 +33,18 @@ qcom_setup_interfaces()
...
+		ucidef_set_interface_wan "eth0"		
+		;;
+
 	yuncore,fap650)
 		ucidef_set_interface_lan "eth3 eth2 eth1 eth0"
 		ucidef_set_interface_wan "eth4"
 		;;
 	wallys,dr6018-v4|\
-	glinet,ax1800)
+	glinet,ax1800|\
```
Comment (blogic) [inline feeds/ipq807x_v5.4/ipq60xx/base-files/etc/board.d/02_network:45]: here aswell

File: feeds/ipq807x_v5.4/ipq60xx/base-files/etc/board.d/02_network [hunk 3]
```diff
@@ -59,8 +64,16 @@ qcom_setup_macs()
 		wan_mac=$(macaddr_canonicalize $mac)
 		lan_mac=$(macaddr_add "$wan_mac" 1)
 		ucidef_set_network_device_mac eth0 $wan_mac
-		ucidef_set_network_device_mac eth1 $lan_mac
-		ucidef_set_label_macaddr $wan_mac
```
Comment (blogic) [inline feeds/ipq807x_v5.4/ipq60xx/base-files/etc/board.d/02_network:63]: and here

File: feeds/ipq807x_v5.4/ipq60xx/base-files/etc/hotplug.d/firmware/10-ath11k-caldata [hunk 4]
```diff
@@ -60,11 +74,11 @@ case "$FIRMWARE" in
 "ath11k/IPQ6018/hw1.0/caldata.bin")
 	case "$board" in
 	cig,wf188n|\
+	cig,wf660a|\
 	edgecore,eap101|\
 	hfcl,ion4xi|\
 	hfcl,ion4x|\
 	hfcl,ion4x_2|\
-	hfcl,ion4x_3|\
 	hfcl,ion4xe|\
```
Comment (blogic) [inline feeds/ipq807x_v5.4/ipq60xx/base-files/etc/hotplug.d/firmware/10-ath11k-caldata:82]: ... and many other places

**Review by blogic — CHANGES_REQUESTED** [review]

**Comment by Kenshi0000** [comment]
John C,
I did NOT removes stuff, this branch was created on Jul 8 based on a449f08839cf65950284d64a6daeb2d35d0ceebd, and today I just had new update on BDF on this branch, basically git should has ability to merge it automatically since I only add the code not delete the code.

**Pull request closed without merging**